src/server/README.md

```diff
@@ -1,5 +1,44 @@
 # Valo-Lineups Server
 
+## Endpoints
+
+All endpoints are read-only endpoints (`GET` requests only), for now we do not need to support user requests to update database values.
+
+All endpoints will begin with the `/api` prefix.
+
+### Maps
+
+| Endpoint                | Description                                    |
+| ----------------------- | ---------------------------------------------- |
+| `/maps`                 | Gets all maps                                  |
+| `/maps/{mapID}`         | Gets a single map corresponding to the `mapID` |
+| `/maps/{mapID}/regions` | Gets all the regions of a map                  |
+
+### Agents
+
+| Endpoint                      | Description                                        |
+| ----------------------------- | -------------------------------------------------- |
+| `/agents`                     | Gets all agents                                    |
+| `/agents/{agentID}`           | Gets a single agent corresponding to the `agentID` |
+| `/agents/{agentID}/abilities` | Gets all the abilities of an agent                 |
+
+### Abilities
+
+| Endpoint                 | Description                                             | Required Params |
+| ------------------------ | ------------------------------------------------------- | --------------- |
+| `/abilities`             | Gets all abilities corresponding to the `agentID` param | `agentID`       |
+| `/abilities/{abilityID}` | Gets a single ability corresponding to the `abilityID`  |                 |
+
+### Markers
+
+| Endpoint                   | Description                                                      | Required Params    | Optional Params         |
+| -------------------------- | ---------------------------------------------------------------- | ------------------ | ----------------------- |
+| `/markers`                 | Gets all markers corresponding to the `mapID` & `agentID` params | `mapID`, `agentID` | `abilityID`, `regionID` |
+| `/markers/{markerID}`      | Gets a single marker corresponding to the `markerID`             |                    |                         |
+| `/markers/{markerID}/info` | Gets the description & media information from the marker         |                    |                         |
```

Comment:
Can we just combine these and just have the marker return coordinates and the info?

Reply:
It might be redundant - I added this as a fallback just in case we decided we wanted to separate the media from the Marker object itself. IIRC however, since these will just be URLs and a relatively short description, it should be fine to just have the one query.

I'll remove next iteration unless someone says otherwise!

Reply:
Per offline conversation, maybe we should split the marker info into to parts, one part being the core marker data when we fetch all markers, and the second being the additional data that will show up after the marker is clicked.